src/database/schema/ChannelListSchema.ts

```diff
@@ -293,7 +293,7 @@ class ChannelList implements BaseDbSchema {
       LEFT JOIN ${UserSchema.getTableName()} B
       ON A.last_updated_by = B.user_id AND A.id = B.channel_id
       WHERE (expired_at IS NULL OR datetime(expired_at) >= datetime('now')) AND A.description != ''
-      AND A.channel_type IN ('ANON_PM', 'ANON_POST_NOTIFICATION', 'ANON_GROUP')
+      AND A.channel_type IN ('ANON_PM', 'ANON_POST_NOTIFICATION', 'ANON_GROUP', 'ANON_TOPIC')
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the SQL query is adding a new channel type 'ANON_TOPIC' to the list of channel types. This seems fine as long as the 'ANON_TOPIC' channel type is valid and expected in your application context. However, it's important to ensure that this new channel type is handled correctly throughout the rest of the application.
```diff
- AND A.channel_type IN ('ANON_PM', 'ANON_POST_NOTIFICATION', 'ANON_GROUP')
+ AND A.channel_type IN ('ANON_PM', 'ANON_POST_NOTIFICATION', 'ANON_GROUP', 'ANON_TOPIC')
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->